src/Tribe/Views/V2/Template_Bootstrap.php

```diff
@@ -344,4 +344,27 @@ public function filter_add_body_classes( $classes ) {
 
 		return $classes;
 	}
+
+	/**
```

Comment:
* :no_entry_sign: Doc comment for parameter "$file" missing
* :no_entry_sign: Doc comment for parameter "$name" missing
* :no_entry_sign: Doc comment for parameter "$template" missing

<sub>hash: 16d3b2545612e654808c1aecfaf9035e</sub>